### PR TITLE
[feat](Nereids) support admin commands

### PR DESCRIPTION
**Pull request by LiBinfeng-01**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

Support admin commands
```ADMIN SHOW REPLICA STATUS FROM baseTableRef wildWhere?                        #adminShowReplicaStatus ShowReplicaStatusStmt*
    | ADMIN COMPACT TABLE baseTableRef wildWhere?                                   #adminCompactTable AdminCompactTableStmt*
```
note that adminCompactTable only supported in syntax, but this feature is not supported when execute, which it would throw an exception when running in backend

### Release note

None

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by LiBinfeng-01** [comment]
run buildall

**Comment by LiBinfeng-01** [comment]
run buildall

**Comment by LiBinfeng-01** [comment]
run cloud_p0

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCompactTableCommand.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import org.apache.doris.nereids.trees.plans.visitor.PlanVisitor;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.StmtExecutor;
+
+import java.util.List;
+
+/**
+ * AdminCompactTableCommand
+ */
+public class AdminCompactTableCommand extends Command {
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCompactTableCommand.java:40]:
```suggestion
public class AdminCompactTableCommand extends Command implements ForwardWithSync {
```

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCompactTableCommand.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    }
+
+    private String getCompactionType() {
+        if (typeFilter == CompactionType.CUMULATIVE) {
+            return "cumulative";
+        } else {
+            return "base";
+        }
+    }
+}
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCompactTableCommand.java:126]: override checkSupportedInCloudMode method and implement as checkDdlStmtSupported method in DdlExecutor.java

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        }
+
+        return true;
+    }
+
+    @Override
+    public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
+        return visitor.visitAdminShowReplicaStatusCommand(this, context);
+    }
+}
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java:154]: override toRedirectStatus method as old planner:
```
@Override
  public RedirectStatus getRedirectStatus() {
      if (ConnectContext.get().getSessionVariable().getForwardToMaster()) {
          return RedirectStatus.FORWARD_NO_SYNC;
      } else {
          return RedirectStatus.NO_FORWARD;
      }
  }
```

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+            return false;
+        }
+
+        if (statusFilter == null) {
+            return false;
+        }
+
+        return true;
+    }
+
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java:149]: override checkSupportedInCloudMode method and implement as checkStmtSupported() method in ShowExecutor.java

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+     */
+    public void validate(ConnectContext ctx) throws AnalysisException, UserException {
+        // check auth
+        if (!Env.getCurrentEnv().getAccessManager().checkGlobalPriv(ConnectContext.get(), PrivPredicate.ADMIN)) {
+            ErrorReport.reportAnalysisException(ErrorCode.ERR_SPECIFIC_ACCESS_DENIED_ERROR, "ADMIN");
+        }
+
+        // bind relation
+        tableRefInfo.analyze(ctx);
+
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java:102]: is Util.prohibitExternalCatalog(tblRef.getName().getCtl(), this.getClass().getSimpleName()); missed?

**Comment by LiBinfeng-01** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCompactTableCommand.java (hunk 1 above)
Comment (LiBinfeng-01) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCompactTableCommand.java]: done

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCompactTableCommand.java (hunk 2 above)
Comment (LiBinfeng-01) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminCompactTableCommand.java:136]: done

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java (hunk 3 above)
Comment (LiBinfeng-01) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java:178]: done

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java (hunk 4 above)
Comment (LiBinfeng-01) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java:156]: done

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java (hunk 5 above)
Comment (LiBinfeng-01) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/AdminShowReplicaStatusCommand.java:109]: done

**Comment by LiBinfeng-01** [comment]
run buildall

**Review by starocean999 — APPROVED** [review]

**Review by englefly — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]
ENV file changes LGTM

**Pull request merged**